features/kinderen.feature

```diff
@@ -136,11 +147,8 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | voornamen | Karel  |
 
   @proxy
-  Rule: Wanneer de geslachtsnaam van het kind onbekend is, wordt het geleverd met type "KindOnbekend" en indicatieOnbekend met waarde true
-    - Dit is het geval wanneer geslachtsnaam dan de standaardwaarde "." heeft
-    - Wanneer geen van de met fields gevraagde kindgegevens een waarde heeft, maar andere gegevens van het kind wel, dan is het type "Kind" en wordt indicatieOnbekend NIET opgenomen
-
-    # Onderliggende aanname is dat wanneer de geslachtsnaam van het kind onbekend is, ook andere kindgegevens niet bekend zijn of niet relevant.
+  Rule: Wanneer een kind alleen "onbekend"-waardes heeft, wordt er een 'kind' zonder gegevens geleverd.
```

Comment:
We leveren geen type Onbekend omdat het lastig is vast te stellen of een kind onbekend is. Om dezelfde reden is ook de nieuwe vorm van deze rule niet terecht. Dus helemaal niks over het kind leveren omdat de geslachtsnaam onbekend is, is niet terecht. Ik zou zeggen, lever wat er is op de manier die we anders ook doen. Dus als de naam wel bekend is en de geboortedatum is onbekend, of de naam is onbekend en de geboortedatum is ook onbekend, dan wordt in beide gevallen op dezelfde manier de onbekende geboortedatum geleverd.

Reply:
Deze opmerking is geborgd in issue #1155